var/spack/repos/builtin/packages/adios/package.py

```diff
@@ -12,30 +36,96 @@ class Adios(Package):
     """
 
     homepage = "http://www.olcf.ornl.gov/center-projects/adios/"
-    url = "https://github.com/ornladios/ADIOS/archive/v1.9.0.tar.gz"
+    url      = "https://github.com/ornladios/ADIOS/archive/v1.10.0.tar.gz"
+
+    version('1.10.0', 'eff450a4c0130479417cfd63186957f3')
```

Comment:
Many packages just `depends_on('autotools', type='build')` and run `autoreconf`.